### PR TITLE
Refactor: remove global rhopw, bigpw, wfcpw, CHR_MIX in module_io

### PR DESCRIPTION
**Pull request by hongriTianqi**

### CODE REVIEW

File: source/module_io/berryphase.cpp [hunk 1]
```diff
@@ -397,8 +401,8 @@ void berryphase::Berry_Phase(int nbands, double &pdl_elec_tot, int &mod_elec_tot
 	
 	for(int istring = 0; istring < total_string; istring++)
 	{
-		phik[istring] = stringPhase(istring,nbands, psi_in, kv);
-		// transfer phase to complex number
+		phik[istring] = stringPhase(istring,nbands, psi_in, rhopw, kv);
+		// 将相位转换成复数形式
```
Comment (Qianruipku) [inline source/module_io/berryphase.cpp]: Why change it?

Comment (hongriTianqi) [reply source/module_io/berryphase.cpp]: This comment is changed back and forth. I should do the job in just one PR as you said. It restores to original state in different PR...

Comment (hongriTianqi) [inline source/module_io/berryphase.cpp]:
```suggestion
        // transfer phase to complex number
```

**Comment by hongriTianqi** [comment]
@Qianruipku I will combine this PR with further works removing GlobalC::wf and GlobalC::wfcpw, please wait a little.

**Review by Qianruipku — APPROVED** [review]

**Pull request merged**